src/main/drivers/dshot.h

```diff
@@ -67,11 +75,21 @@ typedef enum dshotTelemetryType_e {
     DSHOT_TELEMETRY_TYPE_CURRENT        = 3,
     DSHOT_TELEMETRY_TYPE_DEBUG1         = 4,
     DSHOT_TELEMETRY_TYPE_DEBUG2         = 5,
-    DSHOT_TELEMETRY_TYPE_DEBUG3         = 6,
-    DSHOT_TELEMETRY_TYPE_STATE_EVENTS   = 7,
-    DSHOT_TELEMETRY_TYPE_COUNT
+    DSHOT_TELEMETRY_TYPE_STRESS_LEVEL   = 6,
+    DSHOT_TELEMETRY_TYPE_STATUS         = 7,
+    DSHOT_TELEMETRY_TYPE_COUNT          = 8
```

Comment:
Count should get incremented automatically.
```suggestion
    DSHOT_TELEMETRY_TYPE_COUNT
```
-----
Since we are only counting up here, we can actually get rid of all manual assignments in `dshotTelemetryType_t`, like so
```
typedef enum dshotTelemetryType_e {
    DSHOT_TELEMETRY_TYPE_eRPM,
    DSHOT_TELEMETRY_TYPE_TEMPERATURE,
    DSHOT_TELEMETRY_TYPE_VOLTAGE,
    DSHOT_TELEMETRY_TYPE_CURRENT,
    DSHOT_TELEMETRY_TYPE_DEBUG1,
    DSHOT_TELEMETRY_TYPE_DEBUG2,
    DSHOT_TELEMETRY_TYPE_STRESS_LEVEL,
    DSHOT_TELEMETRY_TYPE_STATUS,
    DSHOT_TELEMETRY_TYPE_COUNT
} dshotTelemetryType_t;
```
unless you wanted to imply that the current order of enum elements must not be changed.

Reply:
fixed